src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -468,6 +468,15 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
             });
     }
 
+    private callPageStateServiceMethod(event: PageModelChangeEvent) {
+        console.log('event.type', event.type);
+        if (event.type === PageModelChangeEventType.ADD_CONTENT) {
+            this.dotPageStateService.contentAdded();
```

Comment:
The service could have this method inside, so we can pass the event there and he can add or remove to update the have content. Is cleaner.